test/iron-menu-behavior.html

```diff
@@ -100,6 +113,53 @@
           }, 200);
         });
 
+        test('keyboard events should not bubble', function(done) {
+          var menu = fixture('nested');
+          var keyCounter = 0;
+          var firstNestedMenu = menu.firstElementChild;
+
+          function triggerKeyEvent(keyCode, shift) {
```

Comment:
Can we use MockInteractions for this instead? Or if it's not sufficient, can we update it to include the functionality we need?